### DISCUSSION TITLE
After uodate to 2.18.0 Cannot find module 'sd-notify' Require stack:

### DISCUSSION THREAD

**Discussion by segads** [mKeRix/room-assistant#729] [Q&A]
Hey there
Thanks for room assistant,  brillant work!

, im running a home assistant supervised on ubuntu server 20.04 on a laptop, using the laptop onboard bluetooth room assistant works great with my watch, but after the update I have this error 

[HealthIndicatorService] Could not load sd-notify: Cannot find module 'sd-notify'
Require stack:
- /usr/lib/node_modules/room-assistant/dist/status/health-indicator.service.js
- /usr/lib/node_modules/room-assistant/dist/status/status.controller.js
- /usr/lib/node_modules/room-assistant/dist/status/status.module.js
- /usr/lib/node_modules/room-assistant/dist/app.module.js
- /usr/lib/node_modules/room-assistant/dist/main.js
- /usr/lib/node_modules/room-assistant/bin/room-assistant.js
5/25/2021, 6:34:19 AM - info - IntegrationsModule: Loading integrations: home-assistant, bluetooth-low-energy, bluetooth-classic
5/25/2021, 6:34:19 AM - info - NestFactory

I thought I miss something and I did a fresh home assistant install on ubuntu server and the same error

**Comment by mKeRix** [comment]
You can ignore this error - it's for an optional feature, you shouldn't see that message to begin with. I will fix this in the next version. :)

**Comment by segads** [comment]
Hey thanks to respond,  it's working great now, amazing job, really amazing!
